src/ResourceManager/DataFactories/Commands.DataFactoryV2/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.DataFactory" version="0.2.1-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.DataFactory" version="0.3.0-preview" targetFramework="net452" />
```

Comment:
This needs to be cleaned up.  ActiveDirectory and AUthentication are not required.  If you use WindowsAzure.Storage and its depednecies and the other assemblies, then these need to be included in  the RequiredAssemblies in your psd1 file.  If you do not, then these should be removed form here.

Reply:
Cleaned.